code/datums/components/links_to_operating_computers.dm

```diff
@@ -0,0 +1,123 @@
+/// Anything with this component will show surgeries performed on them
+/// on the operating computer, and optionally provide upgraded surgeries.
+/datum/component/links_to_operating_computers
+	/// Should this object allow the upgraded surgeries?
+	var/provide_upgraded_surgeries = FALSE
+
+	VAR_PRIVATE
+		/// Current known patients, in order of least to most recently added
+		list/patients = list()
+
+/datum/component/links_to_operating_computers/Initialize(provide_upgraded_surgeries)
+	. = ..()
+
+	if (!ismovable(parent))
+		stack_trace("Parent must be a movable")
```

Comment:
This stack trace seems redundant, given returning `COMPONENT_INCOMPATIBLE` throws a stack trace

Reply:
It doesn't tell you why so you have to dig through the code to figure out what specifically it's looking for, which can get more annoying if more conditions are added